_guides/project-boards/configure.md

```diff
@@ -285,11 +287,81 @@ The project board will automatically refresh.  The cards in the completed column
+
+Let's break down how to manually modify each of the sections of code we just blindly copy and pasted into the template editor from earlier:
+
+1. The beginning of each visual cue has some form of an if-statement because we don't want every task to be modified with our visual cue. The if-statement executes if the boolean within it calculates to <i>true</i>. This boolean statement can be on anything (but we suggest you make it related to the task at hand). 
+To that end, you can insert placeholders anywhere in the editor, including inside if-statements. In the sample code above, I only marked tasks overdue if their {due} placeholder was before the current timestamp.
+
+1. After each if-statement there is a body that executes the desired actions if we are in the correct state. All of these actions will change Cerb's default template. These changes can include removing the owner of a task if there is no owner, removing the reopened field when it has never been reopened to begin with, and more. We added a clock icon with red color using the "div" block while also increasing the font to 120% of the default font. The color is modified with traditional RGB coloring. 
+
+<div class="cerb-box note">
+<p>More of the actions available to you are <a href="https://twig.symfony.com/doc/2.x/">here</a> on Twig's site.</p>
+</div>
+
+### Modifying baseline cards for tasks
+
+We are missing some information in our custom template. Let's add the information back Cerb's default template had.
+
+Go ahead and paste this code before line 15:
+
+<pre>
+<code class="language-javascript line-numbers">
+{% raw %}
+&lt;div style="font-size:100%;""&gt;
+    Status: {{status}}
+&lt;/div"&gt;
+&lt;div style="font-size:100%;""&gt;
+    Updated: {{updated|date_pretty}}
+&lt;/div"&gt;
+&lt;div style="font-size:100%;""&gt;
+    Importance: {{importance}}
+&lt;/div"&gt;
+{% if owner_id != "" %}
+&lt;div style="font-size:100%;""&gt;
+    Owner: {{owner__label}}
+&lt;/div"&gt;
+{% endraw %}
+</code>
+</pre>
+
+Now we have all the information back from before. Using the <div> blocks, you can display the information needed for tasks in a specific dashboard. Using the double brackets, placeholders can be called inside of the <div> blocks to access information about the task. Notice how the <b>updated</b> placeholder has a filter applied to it. The output of the placeholder is in seconds, so we needed to use a filter to convert it into a format more human readable. Lastly, we added a few quality of life improvements. Now all of our tasks will show all of our desired information for tasks open and waiting whether they are or are not overdue. Also, tasks without owners will not display a blank owner field like Cerb's default template.
```

Comment:
* Make sure to escape `<div>` with back-ticks.
* Use Markdown formatting for "updated" (code) rather than `<b>` HTML tags.
* This long paragraph could be broken up for readability.